docs/components_page/components/index/simple.jl

```diff
@@ -1,7 +1,20 @@
-using DashBootstrapComponents, DashHtmlComponents
+# 1. Import Dash
+using Dash, DashBootstrapComponents
 
-layout = dbc_alert(
-    "This is an example of a component being used in the wild. " *
-    "Below me, you can find the code used to create me.",
-    color = "info",
+# 2. Create a Dash app instance
+app = dash(external_stylesheets = [dbc_themes.BOOTSTRAP])
+
+# 3. Add the example
+badge = dbc_button(
+    [
+        "Notifications",
+        dbc_badge("4", color = "light", text_color = "primary", className = "ms-1"),
+    ],
+    color = "primary",
 )
+
+# 4. Assign your layout to the app layout
+app.layout = badge
```

Comment:
How about adding a `Container` to the layout for the badge?  And maybe `fluid=True` 
Based on the question on the forum, I think  people new to Bootstrap often miss this step.

Reply:
Good suggestion. Done!